src/utilities/Constants.ts

```diff
@@ -1 +1,5 @@
-export const SPACE_CHANNEL_PREFIX = '_ably_space_';
+const SPACE_CHANNEL_PREFIX = '_ably_space_';
+
+const CURSOR_EVENT = '_event_cursor';
```

Comment:
`_cursor_update`. Also, any particular reason we're prepending these with a `_`? i.e. is this a pattern in the SDK?